packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -730,4 +732,60 @@ export class PluginLedgerConnectorBesu
     );
     return { transaction };
   }
+  public async getBesuRecord(
+    request: BesuRecordLocatorV1Request,
+  ): Promise<BesuRecordLocatorV1Response> {
+    const fnTag = `${this.className}#getBesuRecord()`;
+    this.log.info(request);
+    //////////////////////////////////////////////
+    let abi: any = undefined;
+    const resp: BesuRecordLocatorV1Response = {};
+    const trans = request.transactionHash;
+
+    if (trans != undefined) {
```

Comment:
@AzaharaC Truthiness[1] can be checked by simply putting the variable in the if condition parentheses (see below)

[1]: https://basarat.gitbook.io/typescript/recap/truthy

```suggestion
    if (txHash) {
```